docs/rules/prefer-class-directive.md

```diff
@@ -22,13 +22,16 @@ This rule aims to replace a class with ternary operator with the class directive
 
 ```svelte
 <script>
-  /* eslint svelte/prefer-class-directive: "error" */
+  /* eslint svelte/prefer-class-directive: "empty" */
```

Comment:
I think the configuration is probably wrong.

```suggestion
  /* eslint svelte/prefer-class-directive: ["error", {"prefer": "empty"}] */
```

Reply:
Fixed. Thanks.